tools/cli.js

```diff
@@ -50,26 +100,35 @@ rl.on('close', () => {
   process.exit();
 });
 
-function completer(line) {
-  return [[], line];
-}
-
 const state = {
   client: null,
   connection: null
 };
 
+commandProcessor._complete = (inputs, depth) => {
+  const cmd = inputs[depth];
+  return [completeByFields(cmd, commandProcessor), depth + 1];
+};
+
 commandProcessor.call = (interfaceName, methodName, args, callback) => {
   if (!state.client) return callback(new Error('Not connected'));
   state.connection.callMethod(interfaceName, methodName, args, callback);
 };
 
+commandProcessor.call._help = () => (
+  'call <interfaceName> <methodName> [ <arg> [ , ... ] ]'
+);
+
 commandProcessor.event = (interfaceName, eventName, args, callback) => {
   if (!state.client) return callback(new Error('Not connected'));
   state.connection.emitRemoteEvent(interfaceName, eventName, args);
   callback();
 };
 
+commandProcessor.event._help = () => (
+  'event <interfaceName> <eventName> [ <arg> [ , ... ] ]'
```

Comment:
Events in JSTP don't have a concept of arguments, like methods do. They only have a single argument that can be of any type. Is it just the help that is incorrect here or that's how it is actually implemented so that it's a bug in CLI?